### DISCUSSION TITLE
Add parameter for openai.api_base

### DISCUSSION THREAD

**Discussion by PlutoniaX** [melih-unsal/DemoGPT#37] [Ideas]
Is it possible to add parameter for demoGPT for openai.api_base for allow for custom endpoints?

**Comment by melih-unsal** [comment]
Hi @PlutoniaX,

Apologies for the delay in my response.

Your idea is excellent, and I've immediately noted it for consideration in our next release. Specifically, the suggestions about incorporating openai.api_type, openai.api_version, and openai.api_base are valuable.

Thank you for your contribution to the DemoGPT community.

Warm regards,

Melih.
